content/en/opentelemetry/schema_semantics/host_metadata.md

```diff
@@ -8,10 +8,6 @@ further_reading:
   text: "OpenTelemetry Support in Datadog"
 ---
 
-<div class="alert alert-info">
```

Comment:
I don't think we should remove this one, as the original partner we had for testing this did not end up using the feature, so we have been able to do less validation than we intended

Reply:
@mx-psi  Good to know! Given this, I will add the alert back but call it a "Preview" to align with our new product lifecycle. Let me know if you have any concerns about that approach!

Reply:
That sounds good to me!